net/mwan3/files/etc/hotplug.d/iface/15-mwan3

```diff
@@ -32,7 +32,8 @@ config_get_bool enabled $INTERFACE 'enabled' '0'
 }
 
 if [ "$ACTION" = "ifup" ]; then
-	config_get family $INTERFACE family ipv4
+	config_get familys $INTERFACE family "ipv4 ipv6"
```

Comment:
In the extant code base, "any" is used to indicate both ipv4 and ipv6.

Reply:
@aaronjg  as I can see,
in https://github.com/openwrt/luci/blob/master/applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js#L217,
empty family is "ipv4 + ipv6"

Reply:
The value in "_('XXX')" is the helper text. It actually leaves the field blank. The default value is then "any", as interpreted by fw3. Config files can also explicitly use the 'any' family, eg: https://git.openwrt.org/?p=project/firewall3.git;a=blob;f=helpers.conf;h=b8ae857cb8ce0424eed59e79b876c68048e1b957;hb=HEAD

Which is the same way that mwan3 handles the default in `mwan3_set_user_iptables_rule`
https://github.com/openwrt/packages/blob/master/net/mwan3/files/lib/mwan3/mwan3.sh#L881

Reply:
no, We never set family to the value `'any'` explicitly
and if family is unset or empty, it is dual stack

Reply:
for code in lib/mwan3/mwan3.sh#L881
that is, if we get empty then the default use 'any' as value, 
This is the result required by the shell code itself, not the value of the family itself

Reply:
it is possible to edit the settings using uci or editing the config file directly. The example you showed as a model was from fw3, which can have family be set to `any` in a dual stack setting. mwan3 already supports having family set to any, just like fw3 does. I don't understand why you want to break this.

Reply:
You don't seem to understand?

family in config file is "ipv4" "ipv6" or ""/none
we nevert save family to "any"

Reply:
@ptp52 the line is wrong I agree with @aaronjg 
`config_get familys $INTERFACE family "ipv4 ipv6"`
this should be
`config_get family $INTERFACE family ""`